app/renderer/src/actions/Inspector.js

```diff
@@ -139,6 +143,54 @@ const findElement = _.debounce(async function (strategyMap, dispatch, getState,
   return dispatch({type: SET_INTERACTIONS_NOT_AVAILABLE});
 }, 1000);
 
+const checkErrorsInAction = ({ticks}) => {
+  const errors = [];
+  if (!ticks) {
+    return [i18n.t('gestureEmptyTickError')];
+  }
+
+  for (const tick of ticks) {
+    if (!Object.values(POINTER_TYPES).includes(tick.type)) {
+      errors.push(
+        i18n.t('gestureInvalidEventError', {
+          invalidEvent: tick.type,
+          validEvents: Object.values(POINTER_TYPES).join(', '),
+        }),
+      );
+    } else if (
+      tick.type === POINTER_TYPES.POINTER_MOVE &&
+      typeof tick.button === 'undefined' &&
+      !tick.x &&
+      !tick.y
```

Comment:
I don't think `POINTER_MOVE` even has the `button` property. Did you mean `duration`? Also I think the condition should fail even if only one of the components fails, like so: `tick.type === POINTER_TYPES.POINTER_MOVE && (typeof tick.duration === 'undefined' || !tick.x || !tick.y`?

Reply:
Nice catch. The parameter should be `duration`.